drivers/can/can_loopback.c

```diff
@@ -13,11 +13,33 @@
 #include <logging/log.h>
 LOG_MODULE_DECLARE(can_driver, CONFIG_CAN_LOG_LEVEL);
 
+K_THREAD_STACK_DEFINE(tx_thread_stack,
+		      CONFIG_CAN_LOOPBACK_TX_THREAD_STACK_SIZE);
+struct k_thread tx_thread_data;
+
+struct can_looppback_frame {
+	struct zcan_frame frame;
+	can_tx_callback_t cb;
+	void *cb_arg;
+	struct k_sem *tx_compl;
+};
+
+K_MSGQ_DEFINE(tx_msgq, sizeof(struct can_looppback_frame),
+	      CONFIG_CAN_LOOPBACK_TX_MSGQ_SIZE, 4);
+
 static void dispatch_frame(const struct zcan_frame *frame,
 			   struct can_loopback_filter *filter)
 {
 	struct zcan_frame frame_tmp = *frame;
 
+	LOG_DBG("Receiving %d bytes. Id: 0x%x, ID type: %s %s"
+		, frame->dlc
+		, frame->id_type == CAN_STANDARD_IDENTIFIER ?
+				    frame->std_id : frame->ext_id
+		, frame->id_type == CAN_STANDARD_IDENTIFIER ?
+				    "standard" : "extended"
+		, frame->rtr == CAN_DATAFRAME ? "" : "RTR frame");
+
```

Comment:
It looks a bit weird with the commas at the start of the line. Why not put them at the end of the line like everywhere else?

Reply:
Agree with Henrik, just place the , at the end of the line.

Reply:
Done.